src/IntegrationTests.Core/IncludeMembers.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Linq;
+using AutoMapper.UnitTests;
+using Microsoft.EntityFrameworkCore;
+using Shouldly;
+using Xunit;
+
+namespace AutoMapper.IntegrationTests.Core
+{
+    public class IncludeMembers : AutoMapperSpecBase
+    {
+        class Source
+        {
+            public int Id { get; set; }
+            public string Name { get; set; }
+            public InnerSource InnerSource { get; set; }
+            public OtherInnerSource OtherInnerSource { get; set; }
+        }
+
+        class InnerSource
+        {
+            public int Id { get; set; }
+            public string Name { get; set; }
+            public string Description { get; set; }
+        }
+
+        class OtherInnerSource
+        {
+            public int Id { get; set; }
+            public string Name { get; set; }
+            public string Description { get; set; }
+            public string Title { get; set; }
+        }
+
+        class Destination
+        {
+            public int Id { get; set; }
+            public string Name { get; set; }
+            public string Description { get; set; }
+            public string Title { get; set; }
+        }
+
+        class Context : DbContext
+        {
+            public Context()
+            {
+                Database.EnsureDeleted();
+                Database.EnsureCreated();
+                Seed();
+            }
+
+            protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+            {
+                optionsBuilder.UseInMemoryDatabase("Local");
```

Comment:
Looks good, but this is a problem. Because we want the tests to run with SQL Server, a real database, and we want to configure that in only one place for all the tests. I don't know how one would do that with EF Core, I guess a base context class would work, but maybe there is a better way. In EF6, it's set [here](https://github.com/AutoMapper/AutoMapper/blob/f31e398a596438a939bf19d5bb97ed218124731d/src/IntegrationTests/App.config#L14).

Reply:
Thank you for your review. I will work on changing the setup according to the need.